client/control/components/RecipeList.js

```diff
@@ -169,6 +233,14 @@ class DisconnectedRecipeList extends React.Component {
   }
 }
 
+DisconnectedRecipeList.propTypes = {
```

Comment:
Personally, I prefer to use the `static` keyword, so the propTypes are defined right in the component However, the conventions used for other components uses the `[class].propTypes = {...}` business, so if I used `static` here I'd feel the need to go through and update the rest of the code - but I'd prefer to do that in another branch/PR